tip/environment.py

```diff
@@ -0,0 +1,85 @@
+import os
+import json
+
+from . import packages
+
+
+def _get_environments_dir(tip_dir: str) -> str:
+    """Get path to the directory containing environment files."""
+    environments_dir = os.path.join(tip_dir, "environments")
+    if not os.path.isdir(environments_dir):
+        os.makedirs(environments_dir)
+    return environments_dir
+
+
+def get_environment_path(tip_dir: str, name: str) -> str:
+    """Find file of the environment called `name`."""
```

Comment:
"Find" != "get". I suggest to rename this function to "locate" like in packages.py after renaming the file.